meta/InlineConfigParser.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace dokuwiki\plugin\struct\meta;
+
+/**
+ * Class InlineConfigParser
+ *
+ * Wrapper to convert inline syntax to full before instantiating ConfigParser
+ *
+ * @package dokuwiki\plugin\struct\meta
+ */
+class InlineConfigParser extends ConfigParser {
+
+    /**
+     * Parser constructor.
+     *
+     * parses the given inline configuration
+     *
+     * @param  string  $inline
+     */
+    public function __construct($inline) {
+        // To support a fuller syntax, such as user-specified filters, tokenise here
+
+        // Split into components
+        $components = str_getcsv($inline, '.');
+
+        // Protect against using single quotes
+        foreach ($components as $component) {
+            if ( substr($component, 0, 1) == "'" ) {
+                // Used single quotes rather than double - will need to rerun CSV extraction
+                $enclosure = "'";
+            }
+        }
+        if ($enclosure == "'") $components = str_getcsv($inline, '.', $enclosure);
```

Comment:
And here it gets a bit ugly. As far as I can see the enclosure/quoting is only necessary to support page names containing dots. Using a separator that is not allowed in pages could circumvent this problem. ``{{$page.name>schema>value}}`` could work for example.

Reply:
Good shout; the `>` character gets a bit lost on its own in some fonts, so I'm looking at `->` with `explode()`.

Reply:
All other locations in Struct use the dotted notation, and the ugly bit can be avoided by restricting to one type of quote.